shoop/core/models/orders.py

```diff
@@ -620,4 +619,4 @@ def get_status_display(self):
 
 
 def _round_price(value):
-    return bankers_round(value, settings.SHOOP_ORDER_TOTAL_DECIMALS)
+    return bankers_round(value, 2)
```

Comment:
This probably oughta have some sort of `# TODO: To be fixed in SHOOP-1912` thing, then?